src/components/standalone-link/docs.mdx

```diff
@@ -0,0 +1,133 @@
+---
+componentName: StandaloneLink
+peerComponents:
+  - SwingsetTestIcon
+---
+
+The `StandaloneLink` component should be used when a link is needed outside of a body of text. See more documentation [the HDS Product Figma](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/?node-id=2365%3A21590).
+
+Internally, this component uses two other components:
+
+- `MaybeInternalLink`: to automatically determine whether or not `next/link` should be used
+- `Text`: to ensure conformance font size and weight with the design system
+
+## Props
+
+🚨 Guidance on icon sizes to pass into the `icon` prop:
+
+- If `textSize` is being set to 100 or 200, then size 16 icons are recommended
+- If `textSize` is being set to 300, then size 24 icons are recommended
+
+| Prop name      | Description                                                                      | Required? | Default value | Type                            |
```

Comment:
as mentioned below, please take in account that the `LinkStandalone` component already supports two different colors `primary` and `secondary` (and if I recollect correctly, they're both used in devdot) and likely soon will need to support more of them if it will be used in the `alerts` components.

Reply:
I'm going to double check with the DevDot designers to see if we have both in the designs, I was under the impression we only have the blue ones so far. If we have both, I'll add another prop!

Reply:
Checked with Brayden and we do have a secondary one coming up! Added the `color` prop and an example in https://github.com/hashicorp/dev-portal/pull/100/commits/c62a54cc1d81ddea0e39afed8eb545e1361db93c.